app_feup/lib/view/Pages/login_page_view.dart

```diff
@@ -97,7 +104,7 @@ class _LoginPageViewState extends State<LoginPageView> {
       msg,
       context,
       duration: Toast.LENGTH_LONG,
-      gravity: Toast.BOTTOM,
+      gravity: Toast.TOP,
```

Comment:
Won't this show the toast from the top of the screen then?

Reply:
Yes, this was to avoid that the toast gets hidden by the keyboard. Best solution I think would be enforce to be above the keyboard, but not sure if the package were using for toast messages supports that

Reply:
The package itself might not be able but I'm pretty sure we can do it ourselves. I believe there is a way of checking if the keyboard is opened or not

Reply:
That would be great. The same for the bug report page. I will look into it

Reply:
We have two solutions: look if the window view is shortened (usually by the keyboard) and display the toast on the top if so, or request focus after tapping the login button (and thus hiding the keyboard). I think the latter is a bit better. Wdyt?

Reply:
I agree. Let's make it happen!